cla_backend/apps/checker/utils.py

```diff
@@ -53,8 +53,12 @@ def validate_row(cls, row):
         except Exception:
             raise ValidationError(message=dict(date="Write the date in this format: dd/mm/yyyy"))
         try:
-            assert row[CSV_COL_TIME].strip() in CALLBACK_TIME_SLOTS
-            row[CSV_COL_TIME] = row[CSV_COL_TIME].strip()
+            time = row[CSV_COL_TIME].strip()
+            # Excel will sometimes remove the leading zero from values it interprets as numbers.
+            if time in ["900", "930"]:
```

Comment:
I think if it does not start with 0,1 or 2 then you can safely add the leading 0